lxd/devices.go

```diff
@@ -72,13 +73,18 @@ func deviceNetlinkListener() (chan []string, chan []string, chan device.USBEvent
 			ueventBuf := make([]byte, r)
 			copy(ueventBuf, b)
 			ueventLen := 0
+			udevEvent := false
 			ueventParts := strings.Split(string(ueventBuf), "\x00")
 			props := map[string]string{}
 			for _, part := range ueventParts {
 				if strings.HasPrefix(part, "SEQNUM=") {
 					continue
 				}
 
+				if strings.HasPrefix(part, "libudev") {
```

Comment:
@Lilyjjo this part seems important as it is different from the other uevent event processing logic here, it would be good to have a comment here explaining why this is needed.